### ISSUE TITLE
Show HUD alarm when a change increases or decreases the number tests significantly

### ISSUE THREAD

**Issue by huydhn** [pytorch/test-infra#764]
### Pitch

Now that we have the number of test run recorded on HUD metrics page https://hud.pytorch.org/metrics, the next step is to figure out which change(s) is associated with the increase (or decreases).  Take a concrete example, there were 2 "big" increases in the number of tests on pull on Sep 20th and Sep 21st:

![Screen Shot 2022-09-21 at 15 20 44](https://user-images.githubusercontent.com/475357/191620825-1c963201-eaeb-4f18-9286-028779422763.png)

And it's not trivial to figure out which changes caused those

### Proposed solutions

As the first step, we need an alert indicator on HUD to point out which commits are associated with the changes.  This would be similar to the TTS alarm we have on MiniHud

**Comment by ZainRizvi** [comment]
Can we move that to a rocset query instead?

I'm thinking of the data we need for a query that returned "[timestamp, [PR/commit on trunk], [Number of tests run on it]" already exists in rocset today, right?

If yes, once we have the above results then in the backend it wouldn't take much to find the commits/PRs that corresponded with spikes in test count and we could list them out in a separate view

**Comment by huydhn** [comment]
Yeah, all this information are already in Rockset I think. We would just need to expose it on HUD.

**Comment by huydhn** [comment]
lol, this would be useful to know right away what did this (the drop between 22nd and 23rd):

![Screen Shot 2022-09-23 at 13 41 03](https://user-images.githubusercontent.com/475357/192053236-de482d04-5564-45d1-b606-733c863e2cf3.png)

**Comment by clee2000** [comment]
> lol, this would be useful to know right away what did this (the drop between 22nd and 23rd):

not related to the bigger issue at hand, but this happens to coincide with taking jobs off of circleci

**Comment by ZainRizvi** [comment]
This may be worth treating as a page-able event (during office hours)
